### ISSUE TITLE
Game timer is counting while snake is on 'pause'

### ISSUE THREAD

**Issue by Dayset** [Dayset/Snake#2]
Game timer counts actual app running time.
Even if the app is on hold/pause/frozen.

**Comment by upizs** [comment]
I will gladly fix this.

**Comment by upizs** [comment]
Hi, I have solved the issue, but when trying to push the changed I get 
![image](https://user-images.githubusercontent.com/63521050/172067375-5c334d67-7d71-4d02-a563-4a142004a697.png)
I was wondering if there is something you need to change to give me permission? 
Used git console commands to get a better error message: 
![image](https://user-images.githubusercontent.com/63521050/172067664-c544fbd9-9dc6-4d35-b2f8-571d57d312f6.png)

This is my first time contributing so might be I am doing something wrong. 
Just so you know, I have created a new branch so I am not pushing on to the main branch. 
Thanks

**Comment by Dayset** [comment]
I just added you @upizs to collaborators list.

**Comment by upizs** [comment]
@Dayset Thanks, I will try to push again this evening. It's morning now where I am.

**Comment by upizs** [comment]
@Dayset Thanks, that worked. :)

**Comment by Dayset** [comment]
@upizs I need time to better look at the changes.
Something is not quite right with the pause menu.
Thank you for your contribution to this small tutorial project.

https://user-images.githubusercontent.com/17880263/172228808-b061b653-0295-4c24-9402-9a839d563fcf.mp4

**Comment by upizs** [comment]
@Dayset I see, it didn't clean the screen before starting the game again. Also looked like it stopped the sounds when paused was pressed and didn't continue once unpaused.

**Comment by upizs** [comment]
@Dayset I found where the issue was. The StartGame method could have been started without pressing space, however only when pressed space I cleaned up the text on screen. I have isolated Startgame method inside if statement that listend for space. Now you can only start the game when pressed space. 
You could do the same thing about game over. "Press Space to play again" something like that.

**Comment by Dayset** [comment]
@upizs i will check it out today. Thanks.

**Comment by Dayset** [comment]
Game timer is working perfectly fine now.
Thank you @upizs
